src/app/screens/connectors/ConnectCommando/index.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  HiddenIcon,
+  VisibleIcon,
+} from "@bitcoin-design/bitcoin-icons-react/outline";
+import ConnectorForm from "@components/ConnectorForm";
+import TextField from "@components/form/TextField";
+import ConnectionErrorToast from "@components/toasts/ConnectionErrorToast";
+import * as secp256k1 from "@noble/secp256k1";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import utils from "~/common/lib/utils";
+
+export default function ConnectCommando() {
+  const navigate = useNavigate();
+  const { t } = useTranslation("translation", {
+    keyPrefix: `choose_connector.commando`,
+  });
+  const [formData, setFormData] = useState({
+    host: "",
+    pubkey: "",
+    rune: "",
+    port: 9735,
+    privateKey: generateCommandoPrivateKey(),
+    proxy: "wss://lnproxy.getalby.com",
+  });
+  const [loading, setLoading] = useState(false);
+  const [commandoPrivateKeyVisible, setCommandoPrivateKeyVisible] =
+    useState(false);
+
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value.trim(),
+    });
+  }
+
+  function getConnectorType() {
+    return "commando";
+  }
+
+  function generateCommandoPrivateKey(): string {
+    const privKey = secp256k1.utils.randomPrivateKey();
+    return secp256k1.utils.bytesToHex(privKey);
+  }
+
+  async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    setLoading(true);
+    const host = formData.host;
+    const pubkey = formData.pubkey;
+    const rune = formData.rune;
+    const port = formData.port;
+    const wsProxy = formData.proxy;
+    const privateKey = formData.privateKey;
+    const account = {
+      name: "commando",
+      config: {
+        host,
+        pubkey,
+        rune,
+        port,
+        wsProxy,
+        privateKey,
+      },
+      connector: getConnectorType(),
+    };
+
+    try {
+      let validation;
+      // TODO: for native connectors we currently skip the validation because it is too slow (booting up Tor etc.)
+      if (account.connector === "nativelndhub") {
+        validation = { valid: true, error: "" };
+      } else {
+        validation = await utils.call("validateAccount", account);
+      }
+
```

Comment:
please review this. this contains c&p code. 
can we validate the connection for commando before we save it (can it be initialized easily and fast enough and then unloaded again?